client/hooks/useDestination.js

```diff
@@ -0,0 +1,35 @@
+import { trpc } from '../trpc';
```

Comment:
The purpose of this task is to use the trpc react query wrapper, why are we not using that here ? See the other hooks we have done on the trpc query branch for reference